serial_views.py

```diff
@@ -38,45 +51,120 @@ class Serializer():
     session = Session()
     ##############################################
 
-    def wseries_geometry_view(self):
-        """
-        returns a query result containing a full record from OEP table as GEOJSON
-        featureCollection. Just the geometry is included.
-        All related tables are joined and the values are included as property within the GEOJSON.
-        :return:
-        """
+    # list that stores all query results that are defined as feature object
+    myfeatures = []
+    with open('WAM_APP_FRED/static/WAM_APP_FRED/geodata/germany.geojson', encoding='UTF-8') as g:
+        gj = geojson.load(g)
 
-        features = []
+    def ger_boundaries_view(self):
 
-        with open('WAM_APP_FRED/static/WAM_APP_FRED/geodata/germany.geojson') as f:
-            gj = geojson.load(f)
+        germany_boundaries = Serializer.gj
 
-        print(len(gj['features']))
+        return HttpResponse(dumps(germany_boundaries), content_type="application/json")
 
-        geometry = Point((10.01, 53.57))
-        # geometry = loadswkb(str(record.Series.location.point), True)
-        # feature = Feature(id=record.Series.id, geometry=geometry)
-        feature = Feature(id=101, geometry=geometry)
-        features.append(feature)
-
-        # return HttpResponse(dumps(FeatureCollection(features)), content_type="application/json")
-        return HttpResponse(dumps(gj['features']), content_type="application/json")
-
-    def pp_list_geometry_view(self):
+    def ppr_view(self, request='Berlin'):
         """
```

Comment:
This method is not used, no? There is the function ppr_view defined below. I would suggest to delete the unused code to avoid confusion.